Features/Transfer/Sources/Scenes/ReceiveScene.swift

```diff
@@ -13,24 +13,57 @@ public struct ReceiveScene: View {
     }
 
     public var body: some View {
-        VStack {
+        VStack(spacing: .large) {
+            AssetImageView(assetImage: model.assetModel.assetImage, size: 64)
```

Comment:
```suggestion
            AssetImageView(assetImage: model.assetModel.assetImage, size: 64)
```

add 64 to const / spacing